conda/cli/common.py

```diff
@@ -575,3 +577,21 @@ def disp_env(prefix):
 
     if output:
         print()
+
+
+DEPRECATED = ["force_pscheck", ]
+
+
+def deprecation_warning(func):
+    """Wraps an execute function in a deprecated args checker"""
+
+    msg = "Argument %s is no longer used"
+
+    @wraps(func)
+    def inner(args, parser):
+        for key in DEPRECATED:
+            if not hasattr(args, key):
+                continue
+            warnings.warn(msg % key.replace("_", "-"), DeprecationWarning)
```

Comment:
No, this does not print any output at all to start with.  I like it because it's non-intrusive for now, but if we get to the point that we want to start really pushing it we can tweak what warnings are shown to the user.